cmd/tx.go

```diff
@@ -206,7 +206,15 @@ $ %s tx con demo-path -o 3s`, appName, appName, appName)),
 				return err
 			}
 
-			modified, err := c[src].CreateOpenConnections(c[dst], retries, to)
+			// ensure that the clients exist
+			modified, err := c[src].CreateClients(c[dst])
```

Comment:
Should similar logic exist for `CreateChannel`? Where we create client and connection if they don't exist?

cc: @colin-axner

Reply:
> Where we create client and connection if they don't exist?

Possibly.  The current `tx link` subcommand does that already, FWIW.

Reply:
no preference here, maybe it makes sense to deprecate `create channels`? or rename `tx link`? Doesn't make too much sense to maintain 2 commands that do the same thing

Reply:
opened #400, outside the scope of this pr